docs/user-manual/modules/ROOT/pages/camel-3x-upgrade-guide-3_21.adoc

```diff
@@ -100,6 +100,10 @@ The `certificatePath` parameter can now be specified as resource: for user point
 
 The deprecated option `componentName` was removed. Please use either `producerComponentName` or `consumerComponentName`.
 
+=== camel-vertx-kafka
```

Comment:
The next section actually is for removed components, I think its better to put it there with the others

Reply:
Thx for the feedback, it should be good now